src/test/java/resources/billPay.feature

```diff
@@ -192,4 +192,85 @@ Feature: Bill Payment P2G Test
     And I will sleep for 1000 millisecond
     Then I should not get a response from callback for bill
     And I will sleep for 5000 millisecond
-    Then I should be able to extract response body from callback for bill paid after timeout
\ No newline at end of file
+    Then I should be able to extract response body from callback for bill paid after timeout
+
+
+  @gov
+  Scenario: RTP-002 Request to Pay is unsuccessful because RtP type of Alias is not specified
+    Given I have tenant as "gorilla"
+    And I have a billerId as "GovBill"
+    And I generate clientCorrelationId
+    And I create a new clientCorrelationId
+    Then I can create DTO for Biller RTP Request without alias details
+    And I can call the biller RTP request API with expected status of 400 and "/test" endpoint
+    And I can extract the error from response body and assert the error information
+
+  @gov
+  Scenario: RTP-003 Request to Pay is unsuccessful because RtP type does not match with the information provided
+    Given I have tenant as "gorilla"
+    And I have a billerId as "GovBill"
+    And I generate clientCorrelationId
+    And I create a new clientCorrelationId
+    Then I can create DTO for Biller RTP Request with incorrect rtp information
+    And I can call the biller RTP request API with expected status of 400 and "/test" endpoint
+    And I can extract the error from response body and assert the error information
```

Comment:
you should add the expected message here in scenario
to improve the readability of test case.